core/src/main/resources/hudson/model/CauseAction/summary.jelly

```diff
@@ -24,14 +24,32 @@ THE SOFTWARE.
 
 <?jelly escape-by-default='true'?>
 <j:jelly xmlns:j="jelly:core" xmlns:st="jelly:stapler" xmlns:d="jelly:define" xmlns:l="/lib/layout" xmlns:t="/lib/hudson" xmlns:f="/lib/form">
-  <t:summary icon="orange-square.png">
-    <j:forEach var="entry" items="${it.causeCounts.entrySet()}">
-      <p>
-        <st:include page="description.jelly" it="${entry.key}"/>
-        <j:if test="${entry.value > 1}">
-          <st:nbsp/>${%Ntimes(entry.value)}
-        </j:if>
-      </p>
-    </j:forEach>
-  </t:summary>
+	<t:summary icon="orange-square.png">
+	<h4>${%Cause}</h4>
+	<style type="text/css">
+	div.tab:not(:target) div.content, div.tab:target div.show, div.tab div.hide {
+		display: none;
+	}
+	div.tab:target div.content,  div.tab:target div.hide {
+		display: block;
+	}
+	</style>
+	<div class="tab" id="text1">
+		<div class="show">		
+			<a href="#text1">${%Show}</a>
+			<br> </br>
+			</div>
+		<div class="hide">
+			<a href="#">${%Hide}</a>
+		</div>   
+		<div class="content">
```

Comment:
I figured out that there is only one build cause at my Jenkins jobs -> the build was started by stuff.
And this contains a very huge amount of upstream builds which i want to hide by this feature.

Maybe there can be more than on build cause - but I have never seen something like that. 
If I only allow the hide link, if the causes list is longer than 5, it would be useless for my problem.
My cause list contains only one very large item.
And there is no way to get information about the upstream builds of this build at this special point when showing the yelly.